pkg/runner/cluster_k8s.go

```diff
@@ -98,7 +99,8 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 	}
 
 	var err error
-	_, runenv.TestSubnet, err = net.ParseCIDR("10.33.10.0/24")
+	b := 1 + rand.Intn(200)
```

Comment:
This has to be fixed in a follow up PR.

It is not exactly clear at what point Weave is releasing IP addresses, so even if we remove containers, if we do a second testplan run in the same subnet, we will probably be getting errors right now.

Reply:
Still this is better than just hardcoding a single subnet of length `/24`.

Reply:
Does weave ever release it? If not, I'd say this is strictly worse as things will kind of work till they suddenly start failing.

Is there any way to query weave?

(regardless, we need to at least leave a comment).

Reply:
@Stebalien I agree we should fix this, but unless we have some kind of a work-around, we cannot make progress on any of the other issues, unless we fix this right now.

Reply:
Ultimately there is an issue for this, as currently we don't have a way of selecting the next available data network. We don't really store networks in-use anywhere, and we should do that. I believe on the Docker Swarm runner we query `docker networks ls` and check how many networks are present there - we either need something similar for `weave` or `testground` should be keeping track.

Reply:
Added another issue specifically for this: https://github.com/ipfs/testground/issues/392